sci-electronics/drahnr-oregano/drahnr-oregano-0.84.39.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python3_{4,5,6} )
+PYTHON_REQ_USE='threads(+)'
+
+inherit gnome2-utils python-single-r1 waf-utils xdg-utils
+
+DESCRIPTION="Application for the schematic capturing and simulation of electrical circuits"
+HOMEPAGE="https://github.com/drahnr/oregano"
+SRC_URI="https://github.com/drahnr/oregano/tarball/v${PV} -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+COMMIT="76560fa"
+
+S="${WORKDIR}/${PN}-${COMMIT}"
+
+PATCHES=(
+	"${FILESDIR}/${P}-fix-xdg.patch"
+	"${FILESDIR}/${P}-fix-gschema.patch"
+	)
+
+COMMON_DEP="dev-libs/glib:2
+	dev-libs/libxml2:2
+	x11-libs/goocanvas:2.0
+	x11-libs/gtk+:3
+	x11-libs/gtksourceview:3.0"
+
+DEPEND="${COMMON_DEP}
+	${PYTHON_DEPS}
+	virtual/pkgconfig"
+
+RDEPEND="${COMMON_DEP}
+	|| ( gnome-base/dconf gnome-base/gconf )
+	sci-electronics/electronics-menu"
+#	sci-electronics/ngspice
+#	sci-electronics/gnucap
+
+pkg_setup() {
+	python-single-r1_pkg_setup
+}
+
+pkg_preinst() {
+	gnome2_schemas_savelist
+}
+
+pkg_postinst() {
+	gnome2_icon_cache_update
+	gnome2_schemas_update
+	xdg_desktop_database_update
+	xdg_mimeinfo_database_update
+
+	elog "You'll need to emerge your prefered simulation backend"
+	elog "such as ngspice or gnucap for simulation to work."
+	elog "As an alternative generate a netlist and use sci-electronics/spice"
+	elog "from the command line for simulation."
+}
+
+pkg_postrm() {
+	gnome2_schemas_update
+}
+
+src_configure() {
```

Comment:
wrong functions order, see https://devmanual.gentoo.org/ebuild-writing/functions/index.html for details